docs/faq/help-center.md

```diff
@@ -0,0 +1,30 @@
+---
+sidebar_label: 'Help Center'
+sidebar_position: 5
+---
+
+# Help Center
+
+### How to stay up to date about LUKSO?
```

Comment:
why go from h1 to h3 directly without h2?

Reply:
Yes let's keep h2 so ##, so that we also have a good ToC on the right sidebar